### ISSUE TITLE
Support for begin delimiter and end delimiter?

### ISSUE THREAD

**Issue by xiangnanscu** [guigrpa/docx-templates#66]
Hello, docx-templates is really powerful. It would be really really great if  begin delimiter and end delimiter are support. Something like add `cmdDelimiterBegin` and `cmdDelimiterEnd` in createReport kwargs **and** treat empty string command as `INS`:
```
  const report = await createReport({
    cmdDelimiterBegin: '{',
    cmdDelimiterEnd: '}',
    literalXmlDelimiter: '||',
    processLineBreaks: true,
    noSandbox: false,
    data: {foo:'hello', bar:'world'},})
```
and then in the docx template, write like this:
```
{foo}
{bar}
```
which is equivalent to :
```
{=foo}
{=bar}
```
which is  equivalent to (current version):
```
+++=foo+++
+++=bar+++
```

**Comment by xiangnanscu** [comment]
To be honest, I almost miss this great repo at the first sight due to the `+++` form. As you know, many other template system like DTL or jinja2 in django or lua-resty-template in lua land are using this form. It would provide a familiar syntax for people to quickly get used to this great repo. Also, in simple cases, people just want to render the docx file with a dict of primitives. Clean form is helpful to maintain the template file.

`cmdDelimiterBegin` and `cmdDelimiterEnd` could be limited to same amount chars. i.e. `{{` and `}}` are valid paris while `{{` and `}` not.

**Comment by KAVA-Leigh** [comment]
Totally agree with the above, the only subtractor for docx-templates is the non-standard delimiters. 

It just makes the base template a bit harder to deal with and changing from a commercial product for us, means replacing delimiters in a lot of templates.

**Comment by xiangnanscu** [comment]
@KAVA-Leigh  I create a PR here and test passed. https://github.com/guigrpa/docx-templates/pull/70
The only problem is that I don't know how to build this package. Do you know?

**Comment by vdechef** [comment]
@xiangnanscu look at https://github.com/guigrpa/docx-templates/blob/master/CONTRIBUTE.md

To build the main package just use yarn compile.
Then, go to example packages and run yarn start: with this you can test the library in different environments. For example, in https://github.com/guigrpa/docx-templates/tree/master/packages/example-webpack you can test the library in browser (using webpack)

**Comment by xiangnanscu** [comment]
@vdechef  Thanks. I publish this fork at https://www.npmjs.com/package/docxt in case of the author long time no response. It should be OK but author review is needed.

**Comment by KAVA-Leigh** [comment]
@guigrpa - Just wondering if there any plans to merge the fork by @xiangnanscu ? Thanks

**Comment by guigrpa** [comment]
It's a bit harder to merge the PR now since it includes also other changes (new package name, unrelated improvements, etc.). But I'm going to manually include the changes, since I agree these are important.

I'm sorry it's taken such a long time, but many other things stood between me and OSS ;)

**Comment by guigrpa** [comment]
This is now available as v3.1.0.

@xiangnanscu Thanks a lot for the effort in putting together this PR, and sorry again for the delay!

Note that configuring `cmdDelimiter: ['{', '}']` interferes with GraphQL queries (you could use `[...]` instead). But if you're not using braces in queries or elsewhere in your template, you'll be fine!
